### ISSUE TITLE
V2 Create is unable to create a single-sw flow on WB-5164 under certain conditions

### ISSUE THREAD

**Issue by rtretyak** [telstra/open-kilda#3027]
1. Take WB-5164 novi switch. 
2. Create a flow with `100` bandwidth using V2 API
**Expected:** Flow is successfully created and have UP state
**Actual:** Meter validation error is observed, operation is rolled back and flow has DOWN status
```
Flow 11Dec135516_929_pasta5283 error - Rule validation failed : Rule (ingress) is missing or invalid: switch 00:00:00:90:fb:64:cd:4a, cookie 0x4000000000000001 - UNKNOWN Meter 32 on 00:00:00:90:fb:64:cd:4a have incorrect config - actual value is MeterSchema(switchId=00:00:00:90:fb:64:cd:4a, meterId=MeterId(value=32), flags=[KBPS, BURST, STATS], bands=[MeterSchemaBand(type=1, rate=100, burstSize=99, inaccurate=true)]) while expected value is MeterSchema(switchId=00:00:00:90:fb:64:cd:4a, meterId=MeterId(value=32), flags=[KBPS, BURST, STATS], bands=[MeterSchemaBand(type=1, rate=100, burstSize=100, inaccurate=false)]) (is accurate false, config MeterConfig(id=MeterId(value=32), bandwidth=100))
```

**Comment by andriidovhan** [comment]
fixed in #3039

**Comment by andriidovhan** [comment]
This is issue is covered by autotest(`"Meter burst size is correctly set on Noviflow Wb5164 switches for #flowRate flow rate"`)
